.github/renovate.json

```diff
@@ -3,7 +3,7 @@
     "config:base"
   ],
   "dependencyDashboard": true,
-  "dependencyDashboardApproval": true,
+  "dependencyDashboardApproval": false,
```

Comment:
```suggestion
  "dependencyDashboardApproval": true,
```

Reply:
Also that was added by  @valentijnscholten here https://github.com/DefectDojo/django-DefectDojo/pull/3602/commits/d09d3e3f48ec5f6ad3906a2f63a3b42181f3569b